recipes/tensorboard_plugin_wit/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.6.0" %}
+
+package:
+  name: tensorboard-plugin-wit
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/51/cd/a0c1f9e4582ea64dddf76c1b808b318d01e3b858a51c715bffab1016ecc7/tensorboard_plugin_wit-1.6.0.post3-py3-none-any.whl
```

Comment:
This should use the predictable form of the URL, see the example recipe for how to construct it

Reply:
that's works for sources, but here its a whl file
I found https://github.com/pypa/warehouse/issues/1944, and tried:
https://files.pythonhosted.org/packages/py3-none-any/t/tensorboard-plugin-wit/tensorboard-plugin-wit-1.6.0.post3-py3-none-any.whl
but that does not seem to work either

Reply:
It's the usual PyPI mix up of `-` and `_` being interchangeable except for when they're not:

> https://pypi.io/packages/py3/t/tensorboard-plugin-wit/tensorboard_plugin_wit-1.6.0.post3-py3-none-any.whl

Reply:
cool, thanks